public/app/ui/components/collections/library/list-result.html

```diff
@@ -0,0 +1,4 @@
+<div class="list-result list-result-library">
```

Comment:
This is the per-collection view of each result in "List" mode (similar to NYSL).